compairifuel/lib/src/global/providers/api_provider.dart

```diff
@@ -50,7 +50,8 @@ class ApiService {
       Map<String, String> headers = {
         "Content-Type": "application/json",
```

Comment:
This works as it is now, but after merging the automatic postman branch the content type becomes "x-www-form-urlencoded" instead of json. So this will probably be a seperate PR later

Reply:
Yes, but I will solve it in this PR since it is still one repository, and a breaking change, so this solution is needed to still have a working codebase.

Reply:
This will not be done in this PR but is some that needs to be watched.